sdks/go/test/run_integration_tests.sh

```diff
@@ -118,7 +117,7 @@ test -d sdks/go/test
 command -v docker
 docker -v
 
-if [[ PUSH_CONTAINER_TO_GCR == 'yes' ]]; then
+if [ "$PUSH_CONTAINER_TO_GCR" = "yes" ]; then
```

Comment:
I used double brackets because that's what was used in the rest of the file. We should be consistent.

Reply:
Done